crash/src/app/accident-report-container/upload-images/upload.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="row">
+  <div class="col-8">
+    <label class="btn btn-default p-0">
+      <input type="file" accept="image/*" (change)="selectFile($event)" />
+    </label>
+  </div>
+
+  
+</div>
+
+<div>
+  <img [src]="preview" class="preview" />
+</div>
+
+@if (message) {
```

Comment:
When using conditional ifs in angular, please use *ngIf.

ie: <div *ngIf="message">{{ message }}</div>

Reply:
It looks like this is a new way of doing *ngIf from Angular 16, please ignore this comment